### PR TITLE
Redesign start and tools page

### PR DESCRIPTION
**Pull request by t-sommer**

### CODE REVIEW

**Comment by chrbertsch** [comment]
FMI-Design Meeting :

- @t-sommer: could you please attach a zip file of the generated webpages for convenience for those who do not have Docker running to create them?
- regarding use of logo: ask too vendors provide change or approve the logos themselves
- publish soon and then improve!

**Comment by t-sommer** [comment]
[_site folder w/o large assets](https://github.com/modelica/fmi-standard.org/files/9053675/_site.zip)

**Comment by t-sommer** [comment]
> * could you please attach a zip file of the generated webpages

The generated files have to be served through a web server to be displayed correctly.

**Comment by t-sommer** [comment]
I've regenerated the `assets/tools.json` from the latest (and frozen) `_data/tools.csv`. Everyone can now start to push their changes and add their logos.

**Comment by chrbertsch** [comment]
For your convenience I have attached some screenshots:
Bildschirmfoto von 2022-07-11 21-54-16
Bildschirmfoto von 2022-07-11 21-48-27
Bildschirmfoto von 2022-07-11 21-55-39
Bildschirmfoto von 2022-07-11 21-56-18
Bildschirmfoto von 2022-07-11 21-57-23
Bildschirmfoto von 2022-07-11 21-57-29




This is a good start and after aprovement of the Steering committee we can iterate to improve.

**Comment by chrbertsch** [comment]
@t-sommer : could you please describe where and how to provide the logo? (--> /assets/images, also provided via a PR ).

And something like: "_By submission of the logo the submitter agrees to the usage of the logo on the tools webpage (in the name of the  logo / trademark  owner)"_

**Review by chrbertsch — CHANGES_REQUESTED** [review]
Tool vendors must add their logo on their own and thus consent to its usage. 
I someone adds a logo for a different company (as has been done e.g. for dSpace), it must be documented, who of this company consented to its usage.

File: assets/tools.json [hunk 1]
```diff
@@ -0,0 +1,3667 @@
...
+        ],
+        "fmuImport": []
+    },
+    {
+        "name": "Activate",
+        "license": "commercial",
+        "url": "https://solidthinking.com/product/activate",
+        "vendor": "Altair",
+        "vendorURL": "https://www.altair.com/",
+        "description": "Software environment for modeling, simulation and analysis of multi-disciplinary systems from Altair",
```
Comment (masoud-najafi) [inline assets/tools.json:137]: {
        "name": "Altair Activate",
        "license": "commercial",
        "url": "https://www.altair.com/activate",
        "logo": "icon_Activate.png"
        "vendor": "Altair",
        "vendorURL": "https://www.altair.com/",
        "description": "Software environment for modeling, simulation and analysis of multi-disciplinary systems from Altair",
        "features": [],
        "platforms": [
            "Linux",
            "Windows"
        ],
        "fmiVersions": [
            "1.0",
            "2.0",
            "3.0"
        ],
        "fmuExport": [
            "CS",
            "ME"
        ],
        "fmuImport": [
            "CS",
            "ME"
        ]
    },
    {
        "name": "Altair MotionSolve",
        "license": "commercial",
        "url": "https://www.altair.com/motionsolve/",
        "vendor": "Altair",
        "logo": "icon_MotionSolve.png"
        "vendorURL": "https://www.altair.com/",
        "description": "MotionSolve is a solver that performs 3D multi-body system simulations to predict the dynamic response and optimize the performance of products that move.",
        "features": [],
        "platforms": [
            "Linux",
            "Windows"
        ],
        "fmiVersions": [
            "2.0"
        ],
        "fmuExport": [
            "CS"
        ],
        "fmuImport": [
            "CS",
            "ME"
        ]
    },
    {
        "name": "Altair MotionView",
        "license": "commercial",
        "url": "https://www.altair.com/motionsolve/motionview/",
        "logo": "icon_MotionView.png"
        "vendor": "Altair",
        "vendorURL": "https://www.altair.com/",
        "description": "MotionView is a graphical environment for building multibody system models and for visualizing motion simulation results using animations and plots.",
        "features": [],
        "platforms": [
            "Linux",
            "Windows"
        ],
        "fmiVersions": [
            "2.0"
        ],
        "fmuExport": [
            "CS"
        ],
        "fmuImport": [
            "CS",
            "ME"
        ]
    },
icon_Activate
icon_MotionSolve
icon_MotionView

**Review by masoud-najafi — COMMENTED** [review]
Hi,
I updated the information for Altair-Activate, Altair-MotionSolve and altair-MotionView.
I also added the logo for these tools.

File: assets/tools.json [hunk 2]
```diff
@@ -0,0 +1,3671 @@
...
+        "fmuExport": [
+            "CS",
+            "ME"
+        ],
+        "fmuImport": [
+            "CS",
+            "ME"
+        ]
+    },
+    {
```
Comment (MBlesken) [inline assets/tools.json:883]: Would you please update for dspace tools:
{
    "name": "ConfigurationDesk",
    "license": "commercial",
    "url": "https://www.dspace.com/go/configurationdesk",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "ConfigurationDesk is an intuitive, graphical configuration and implementation tool, ideal for handling applications from small Rapid Control Prototyping (RCP) developments to large Hardware-in-the-Loop (HIL) tests based on dSPACE real-time hardware, such as SCALEXIO or MicroAutoBox III, including the implementation of behavior models and I/O function code.",
    "features": [],
    "platforms": [
      "Linux"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "MicroAutoBox III",
    "license": "commercial",
    "url": "https://www.dspace.com/go/microautobox3",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "MicroAutoBox III is a real-time simulation system for performing fast, in-vehicle function prototyping. Simulations for MicroAutoBox III are implemented with ConfigurationDesk.",
    "features": [],
    "platforms": [
      "Linux"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "SCALEXIO",
    "license": "commercial",
    "url": "https://www.dspace.com/go/scalexio",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "SCALEXIO is a Hardware-in-the-Loop (HIL) integration and simulation platform from dSPACE. Simulations for SCALEXIO are implemented with ConfigurationDesk.",
    "features": [],
    "platforms": [
      "Linux"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "SYNECT",
    "license": "commercial",
    "url": "https://www.dspace.com/go/synect",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "SYNECT is a data management tool from dSPACE that enables you to manage FMUs and Simulink models as well as their dependencies, versions and variants throughout the entire software development process.",
    "features": [],
    "platforms": [
      "Linux",
      "Windows"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [],
    "fmuImport": [
      "CS"
    ]
  },
  {
    "name": "SystemDesk",
    "license": "commercial",
    "url": "https://www.dspace.com/go/systemdesk",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "SystemDesk is an environment for creating virtual ECUs for SIL simulation. Virtual ECUs can be on application level or they can contain up to the complete hardware-independent ECU software.",
    "features": [],
    "platforms": [
      "Windows"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [
      "CS"
    ],
    "fmuImport": []
  },
  {
    "name": "TargetLink",
    "license": "commercial",
    "url": "https://www.dspace.com/go/targetlink",
    "logo": "dSPACE.png",
    "vendor": "dSPACE",
    "vendorURL": "https://www.dspace.com/go/fmi",
    "description": "TargetLink is a production code generator from dSPACE that generates highly efficient C-code directly from Simulink/Stateflow models.",
    "features": [],
    "platforms": [
      "Linux",
      "Windows"
    ],
    "fmiVersions": [
      "2.0"
    ],
    "fmuExport": [
      "CS"
    ],
    "fmuImport": []
  },
    {
      "name": "VEOS",
      "license": "commercial",
      "url": "https://www.dspace.com/go/fmi",
      "logo": "dSPACE.png",
      "vendor": "dSPACE",
      "vendorURL": "https://www.dspace.com/go/fmi",
      "description": "VEOS is a PC-based virtual integration and simulation platform from dSPACE.",
      "features": [],
      "platforms": [
        "Linux",
        "Windows"
      ],
      "fmiVersions": [
        "1.0",
        "2.0"
      ],
      "fmuExport": [],
      "fmuImport": [
        "CS"
      ]
    },

**Review by MBlesken — CHANGES_REQUESTED** [review]
Changes requested for dspace tools (one tool added)

Questions:
* I assumed that "platforms" lists the platforms the respective (i.e. generated or managed) FMUs are compatible with, and does not refer to the host system the tool can be run on. Is that right? Should it be stated more explicitly on the tools page?
*  Do you see need to explicitly state compatibility for source FMUs? That's not possible yet, or did I miss it?
* Should it be possible to mark that a support of FMI 3 is planned for?

FYI @t-sommer @chrbertsch

**Comment by TorstenBlochwitz** [comment]
Do we really want to include tool-logos to the tools-list? 
Please consider the effort for:
- maintenance (logos change from time to time)
- design (logos should have similar size to treat all tools visually equal)
From my point of view, we should minimize the amount of information on this page. The tool-link can link to a web-site with detailed information including logos.

**Comment by chrbertsch** [comment]
> * maintenance (logos change from time to time)

Committing and updating the tool logos is up to the tool vendors (also for legal reasons). Thus maintenance effort from our side is minimal.

I like the logos.

**Review by chrbertsch — APPROVED** [review]

**Pull request merged**